### PR TITLE
Remove unnecessary asNumber logic

### PR DESCRIPTION
**Pull request by epicfaace**
This is for PR #1183. I believe we don't need this logic anymore, with the changes implemented in the aforementioned PR.

### CODE REVIEW

**Comment by glasserc** [comment]
This looks like it got a bad merge somehow? I think you just intended to include the last commit.

**Comment by LucianBuzzo** [comment]
@epicfaace Can you rebase this PR on top of master please?

**Comment by epicfaace** [comment]
@LucianBuzzo done!

File: packages/core/test/utils_test.js [hunk 1]
```diff
@@ -1152,12 +1152,11 @@ describe("utils", () => {
     });
 
     it("should return the raw value if the input ends with a dot", () => {
```
Comment (epicfaace) [inline packages/core/test/utils_test.js]:
```suggestion
    it("should convert the value to an integer if the input ends with a dot", () => {
```

**Pull request closed without merging**